Donations page styling

yi-json opened the pull request:
Added slider functionality on the Donations page. Cropped images, and added whitespace and round corners on images. Implemented mobile compatability.

yi-json committed:
Added slideshow on Donations page

yi-json committed:
Added mobile compatability, margins, and padding

nikhilabraham1 committed:
cropped images

nirmalunc reviewed (changes requested):
Seems to have issues with mobile compatibility, please review and fix. Looks great on desktop!

jackbuoy committed:
Fix mobile compatability for donations page
Changed width of images and mobile section. Added classNames to the google forms so they could be styled for mobile. Made small changes to a few of the mobile classes for better visibility on mobile.

nirmalunc committed:
Minor text styling update to donations descriptions

nirmalunc committed:
Merge branch 'master' into donationsPageStyling

nirmalunc reviewed (approved):
LGTM!
